command/operator_debug.go

```diff
@@ -365,6 +379,8 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	flags.StringVar(&interval, "interval", "30s", "")
 	flags.StringVar(&c.logLevel, "log-level", "TRACE", "")
 	flags.BoolVar(&c.logIncludeLocation, "log-include-location", true, "")
+	flags.StringVar(&logLookback, "log-lookback", "", "")
+	flags.StringVar(&logFileExport, "log-file-export", "", "")
```

Comment:
Shouldn't `log-file-export` be a bool flag? This would require that we set `-log-file-export=true` and not just have `-log-file-export` set.